src/apps/inventory/migrations/0003_product_created_at.py

```diff
@@ -1,7 +1,7 @@
 # Generated by Django 4.2.5 on 2023-09-29 13:40
 
-from django.db import migrations, models
```

Comment:
Why?

Reply:
done by isort

Reply:
no need to do that in migrations

and especially in files that not yours